nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core/Dependency.cs

```diff
@@ -1,3 +1,3 @@
 namespace NuGetUpdater.Core;
 
-public sealed record Dependency(string Name, string? Version, DependencyType Type, bool IsDevDependency = false, bool IsOverride = false);
+public sealed record Dependency(string Name, string? Version, DependencyType Type, bool IsDevDependency = false, bool IsOverride = false, bool UpdateOnly = false);
```

Comment:
Suggestion, rename to be consistent with other flags that are of the form 'IsX'.